packages/react-components/src/components/VerticalGallery.tsx

```diff
@@ -210,13 +211,21 @@ const VerticalGalleryControlBar = (props: VerticalGalleryControlBarProps): JSX.E
 
   const controlBarSpacing = { childrenGap: '0.5rem' };
 
+  const previousButtonAriaLabel = strings.leftNavButtonAriaLabel
+    ? _formatString(strings.leftNavButtonAriaLabel, { current: `${currentPage}`, total: `${totalPages}` })
```

Comment:
I wonder if this will be okay languages that don't use numeric numbers (0-9) like Arabic.